cmake/UraniumTranslationTools.cmake

```diff
@@ -108,6 +108,6 @@ if(GETTEXT_FOUND)
         TARGETS_FOR_PO_FILES(${language})
         TARGETS_FOR_MO_FILES(${language})
     endforeach()
-    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium/)
+    install(DIRECTORY ${CMAKE_BINARY_DIR}/resources DESTINATION ${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}/)
```

Comment:
No, since Uranium in cura-build is built by calling an external CMake, not included in the main project. But I see now that CMake also defines PROJECT_NAME... Yay consistency!